### PR TITLE
[v0.5.0] Add Locust tutorial

### PR DESCRIPTION
**Pull request by eduardogmisiuk**
Related to https://github.com/dojot/dojot/issues/1473

### CODE REVIEW

File: source/load-test-with-locust.rst [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+
+
+General tips for the test
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Ansible has the ``100k`` tag to prepare the environment for a 100,000 clients test. It runs a
+  minified version of dojot without some services. This was made because not all services are ready
+  yet to support such a load.
+- By using volumes in EJBCA pods, you can scale them up to increase the certificate creation
+  throughput.
```
Comment (thiagodpf) [inline source/load-test-with-locust.rst:456]: Suggestion:
> Sharing a persistent volume between EJBCA Pods, [...]

File: source/locale/pt_BR/LC_MESSAGES/load-test-with-locust.po [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+msgid "Enter in Locust directory:"
+msgstr "Entre na pasta do Locust:"
+
+#: ../../source/load-test-with-locust.rst:36
+msgid "Running a simple test"
+msgstr "Rodando um teste simples"
+
+#: ../../source/load-test-with-locust.rst:38
+msgid "In this section, will be shown how to configure and run Locust and generate certs script in order to execute the load test. This is a simple test where will be created 100 clients to send messages to dojot."
+msgstr "Nesta seção, será demonstrado como configurar e rodar o Locust, além de como gerar certificados com o script ``generate_certs`` que serão utilizados para rodar o teste. Este é um teste teste em que serão criados 100 clientes que enviarão mensagens à dojot."
```
Comment (thiagodpf) [inline source/locale/pt_BR/LC_MESSAGES/load-test-with-locust.po:60]: > [...] que serão utilizados para rodar o teste. Este é um **teste teste** em que serão criados [...]

File: source/locale/pt_BR/LC_MESSAGES/load-test-with-locust.po [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+msgid "Generating the certificates"
+msgstr "Gerando os certificados"
+
+#: ../../source/load-test-with-locust.rst:93
+msgid "As said before, the communication from Locust to dojot is secure, so there is the need to create certificates."
+msgstr "Como dito anteriormente, a comunicação entre Locust e dojot é segura, portanto é necessário o uso de certificados."
+
+#: ../../source/load-test-with-locust.rst:96
+msgid "There are two ways of simulating devices: you can create fake devices (will not show up in dojot's GUI) or real devices. In this part of the tutorial, we will create real devices, so you can check the sent messages in the GUI."
+msgstr "Existem duas maneiras de se simular os dispositivos: você pode criar dispositivos falsos (que não aparecerão na GUI da dojot) or dispositivos reais. Nesta parte do tutorial serão criados os dispositivos reais para que você possa verificar a chegada das mensagens na GUI."
```
Comment (thiagodpf) [inline source/locale/pt_BR/LC_MESSAGES/load-test-with-locust.po:102]: > [...] GUI da dojot) **or** dispositivos reais. [...]

**Review by thiagodpf — COMMENTED** [review]
The documentation is very good, I could understand everything. I left just a few notes.

**Review by thiagodpf — APPROVED** [review]
:100:

File: source/load-test-with-locust.rst [hunk 4]
```diff
@@ -0,0 +1,479 @@
+Load test using Locust
```
Comment (mprevide) [inline source/load-test-with-locust.rst:1]: Maybe you can put a more generic title like Using load test to measure the environment supports, our something better...

File: source/load-test-with-locust.rst [hunk 5]
```diff
@@ -0,0 +1,479 @@
...
+
+
+Running the test
+^^^^^^^^^^^^^^^^
+
+You are all set to begin the test. To run it, you need to access the Locust interface in your
+browser in ``localhost:8089`` (supposing you are running Locust master in ``localhost``).
+
+Type ``100`` in ``Number of users to simulate``, ``10`` in ``Hatch rate`` and click in
+``Start swarming``. This tells Locust to run 100 clients, creating 10 of them per second.
```
Comment (mprevide) [inline source/load-test-with-locust.rst:168]: This tells Locust to run 100 devices, publish 10  messages per second. ?

Comment (eduardogmisiuk) [reply source/load-test-with-locust.rst:168]: No, the `Hatch rate` is the speed at which Locust will create clients. The number of messages per second depends on an environment variable in the slave docker compose file.

File: source/load-test-with-locust.rst [hunk 6]
```diff
@@ -0,0 +1,479 @@
...
+master log a message for each slave that connects to it.
+
+Running the distributed test
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We are all set to begin the test. To run it, you need to access the Locust interface in your browser
+in the Locust master location, e.g.: ``localhost:8089``.
+
+Type ``1000`` in ``Number of users to simulate``, ``10`` in ``Hatch rate`` and click in
+``Start swarming``. This tells Locust to run 1,000 clients, creating 10 of them per second.
```
Comment (mprevide) [inline source/load-test-with-locust.rst:346]: This tells Locust to run 1000 devices, publish 10 messages per second. ?

File: source/load-test-with-locust.rst (hunk 4 above)
Comment (eduardogmisiuk) [reply source/load-test-with-locust.rst:1]: By mentioning load test, doesn't it already mean that we are measuring the environment support?

Comment (mprevide) [reply source/load-test-with-locust.rst:1]: I think I might not mention "locust" in the title, but something more meaningful for someone who doesn't know "locust". And also why this page could be interesting for anyone who is going to read.

Comment (eduardogmisiuk) [reply source/load-test-with-locust.rst:1]: Fair enough!

**Review by mprevide — APPROVED** [review]
:1st_place_medal:

**Review by thiagodpf — APPROVED** [review]
:+1:

**Pull request merged**